applications/swarm-nl.md

```diff
@@ -0,0 +1,156 @@
+# SwarmNL
+- **Team Name:** Algorealm, Inc.
+- **Payment Address:** TBA
+- **Level:** 2
+
+## Project Overview :page_facing_up:
+
+### Overview
+
+SwarmNL is a configurable P2P networking layer designed to be used in distributed system architectures that require data transfer solutions for off-chain communication in a scalable way.
+
+It's goal is to provide a lightweight and configurable P2P networking layer for web applications designed to interact with on-chain data.
```

Comment:
What we meant is that it could be extended to serve as the networking stack for on-chain data transfer. Of course, there will be a need for a subsystem for verification etc. But with what we envision for SwarmNL, adding things like that would not have too much overhead. 
Maybe we should make that correction and add it as something that COULD be done, or remove it altogether.